glossary.md

```diff
@@ -39,3 +39,21 @@ USA -- typeOf ------------> Country
 USA -- name --------------> United States of America
 USA -- containedInPlace --> northamerica
 ```
+
+- **[Measurement method](https://datacommons.org/browser/measurementMethod)**
+
+The technique used for measuring a statistical variable. Describes how a measurement is made, whether by count or estimate or some other approach. May name the group making the measurement to indicate a certain organizational method of measurement is used. Examples include `BosniaCensus` and [`WorldHealthOrganizationEstimates`](https://datacommons.org/browser/WorldHealthOrganizationEstimates).
+
+- **[Observation period](https://datacommons.org/browser/observationPeriod)**
+
+The time period over which an observation is made.
+
+- **[Unit](https://datacommons.org/browser/unit)**
+
+The unit of measurement. Examples include Kelvin, Celsius, inches, light years, and slugs.
+
+- **[Scaling factor](https://datacommons.org/browser/scalingFactor)**
+
+Property of statistical variables that measure proportions, used in conjunction with the measurementDenominator property to indicate the multiplication factor applied to the proportion's denominator (with the measurement value as the final result of the multiplication) when the numerator and denominator are not equal.
+
+As an example, in 1999, [approximately 36% of Canadians were Internet users](https://datacommons.org/browser/dc/o/2mthzyv99kd73). Here the measured value of `Count_Person_IsInternetUser_PerCapita` is 36, and the scaling factor or denominator for this per capita measurement is 100. Without the scaling factor, we would interpret the value to be 36/1, or 3600%.
```

Comment:
The example is great! I think the first paragraph makes sense, but possibly bc I already understand scalingFactor. I might suggest bringing the first paragraph to a semi/technical friend and see if it is helpful for them to understand.

Nit: could we add measurementDenominator to the glossary too?

Reply:
I talked to my sister (who is a nursing major, but with three siblings and two parents in tech) about this definition, and she said that while she didn't really understand the statistical variables terminology, it made sense to her what a scaling factor was.

I will add measurementDenominator as well.